.github/workflows/main.yml

```diff
@@ -102,4 +102,4 @@ jobs:
         esac
         . "${VENV_DIR}/${scripts}/activate"
         echo "Using $(python --version) on ${{ matrix.os }}" 1>&2
-        nosetests --traverse-namespace --with-coverage tests
+        nose2 -v --with-coverage tests
```

Comment:
`-v` may not be necessary, but it was helpful to make sure things were running. 
I don't know what `--traverse-namespace` does, but it is not supported in nose2